numpy/random/tests/test_generator_mt19937_regressions.py

```diff
@@ -132,7 +131,7 @@ def test_permutation_subclass(self):
         class N(np.ndarray):
             pass
 
-        mt19937.bit_generator.seed(1)
+        mt19937 = RandomState(1)
```

Comment:
Why is this test using `RandomState` while all of the others using `Generator`?

Reply:
Removing last vestiges of RandomState from this file.